docs/generate_from_k8s_spec.md

```diff
@@ -0,0 +1,31 @@
+# Generate KCL Packages from Kubernetes OpenAPI Specs
+
+This guide shows how the `kusion_kubernetes` packages are generated. Alternatively, you could skip this guide and directly use the pre-generated Kubernetes packages which are exactly the same as the outcome of this guide. Please refer the [kpm quick start guide](https://github.com/kcl-lang/kpm#quick-start) for how to pull and use the package.
+
+If you want to manually generate them, please continue the guide.
+
+## 1. Download the OpenAPI Spec
+
+Download the Kubernetes OpenAPI Spec from Github. In this guide, we'll generate from the [Kubernetes v1.27 spec](https://github.com/kubernetes/kubernetes/blob/release-1.27/api/openapi-spec/swagger.json)
+
+## 2. Pre-process the Spec
+
+Download the pre-process script from [here](../scripts/preprocess/main.py), then run:
+
+```shell
+export spec_path="<path to the Kubernetes openAPI Spec>"
+export script_path="path to main.py"
+python3 ${script_path} ${spec_path} --omit-status --rename=io.k8s=kusion_kubernetes
```

Comment:
What is the expected execution result of this command?

Reply:
a processed k8s openapi spec file.
For example, after processing file `a/b/c.json`, the `a/b/processed-c.json` will be output